### PR TITLE
Fix incorrect menu/link on webhook edit page

### LINKED ISSUE #29699: Incorrect menu/link on webhook edit page
Maybe a regression, invalid link (or incorrect menu):

image

_Originally posted by @wxiaoguang in https://github.com/go-gitea/gitea/issues/29114#issuecomment-1986832261_

### PR DESCRIPTION
**Pull request by yp05327**
Fix #29699

### CODE REVIEW

**Comment by lunny** [comment]
Can you add some tests to avoid the next possible broken?

**Review by delvh — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by yp05327** [comment]
> Can you add some tests to avoid the next possible broken?

Done.

**Comment by silverwind** [comment]
Needs `make fmt`

File: tests/integration/repo_webhook_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// SPDX-License-Identifier: MIT
+
+package integration
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+
+	"code.gitea.io/gitea/tests"
```
Comment (silverwind) [inline tests/integration/repo_webhook_test.go:11]:
```suggestion
	"code.gitea.io/gitea/tests"

```

**Pull request merged**